src/views/CoCurricularTranscript/index.tsx

```diff
@@ -36,14 +44,155 @@ const CoCurricularTranscript = () => {
     loadTranscript();
   }, [profile]);
 
-  if (loading || loadingProfile) {
+  if (loadingProfile) {
     return <GordonLoader />;
   }
 
   if (!profile) {
-    return <GordonUnauthorized feature={'your experience transcript'} />;
+    const participation1: Participation = Participation.Member;
+    const session1: MembershipHistorySession = {
+      MembershipID: 6547,
+      SessionCode: '202306',
```

Comment:
Yes, but that won't look good in a few years since it will be old data, we'll need task to maintain it. I would recommend connecting it to the current term rather than a specific one.

Reply:
Oh good idea. Will do!